android/app/build.gradle

```diff
@@ -211,7 +212,7 @@ dependencies {
     implementation "com.android.support:appcompat-v7:26.1.0"
     // Force using exact RN version instead of relying on gradle dependency resolution
     // https://docs.gradle.org/current/userguide/introduction_dependency_management.html#sec:dependency_resolution
-    compile ("com.facebook.react:react-native:0.55.4") { force = true }  // From node_modules
```

Comment:
Previous comment applies to the use of `compile` I believe. @dmitryn Can you comment?